chore(CPlusPlus): add armstrong

kumarsyadav2 opened the pull request:
<!--
Please make sure to read the Contribution Guidelines:
-->

<!-- PULL REQUEST TEMPLATE -->
<!-- (Update "[ ]" to "[x]" to check a box) -->

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [x] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

<!-- For example you can add algorithm name, language used to implement it and link to online implementation of it -->

**Other information:**
<!-- Add any additional info you want here -->

Ashborn-SM commented on algorithms/CPlusPlus/Maths/armstrong.cpp:
```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+int main() {
+int num, i=0,r,sum=0,s;
+//cout <<"enter a number to check it is Armstrong no. or not ";
+cin>>num; 
+s=num;
+while (i<num)
+{
+    r=num%10;
+    sum =sum+(r*r*r);
+    num =num/10;
+}
+if(sum==s)
+cout <<"it is a Armstrong number ";
+else 
+cout <<"it is not a Armstrong number ";
+    return 1;
+}
```
please give proper indentation
```suggestion
    int num, i=0,r,sum=0,s;
    //cout <<"enter a number to check it is Armstrong no. or not ";
    cin>>num; 
    s=num;
    while (i<num)
    {
        r=num%10;
        sum =sum+(r*r*r);
        num =num/10;
    }
    if(sum==s)
    cout <<"it is a Armstrong number ";
    else 
    cout <<"it is not a Armstrong number ";
    return 1;
}
```

Ashborn-SM commented on algorithms/CPlusPlus/Maths/armstrong.cpp:
```diff
@@ -0,0 +1,19 @@
+//cout <<"enter a number to check it is Armstrong no. or not ";
+cin>>num; 
+s=num;
+while (i<num)
+{
+    r=num%10;
+    sum =sum+(r*r*r);
+    num =num/10;
+}
```
This program is only valid for 3 digit number.
What if i give a 4 digit number or more?

kumarsyadav2 replied:
Ok, let me correct it, so that it also suits to big numbers

ming-tsai commented:
Hi @Arsenic-ATG,
Could you check this pull request?

Arsenic-ATG commented on algorithms/CPlusPlus/Maths/armstrong.cpp:
```diff
@@ -0,0 +1,19 @@
+    while (i<num)
+    {
+        r=num%10;
+        sum =sum+(r*r*r);
```
As @Ashborn-SM already pointed out, the program doesn't work for numbers with less or more digits than a three digit number.

By definition, An [Armstrong number ( or Narcissistic_number )](https://en.wikipedia.org/wiki/Narcissistic_number) is a number that is the sum of its own digits each raised to the power of the **number of digits**.

Arsenic-ATG commented:
looks like there are still unresolved conflicts present in the your file.

Arsenic-ATG commented:
Also update the algorithm/readme.md file with your creation in it.

kumarsyadav2 commented:
I have corrected the armstrong code and also updated read me file, kindly checkout these changes @member and do let me know where it is eligible to getting merge into it or not.
